dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/ingest/IngestServiceBean.java

```diff
@@ -155,6 +155,7 @@
     // DataFileCategory objects, if any were already assigned to the files).
     // It must be called before we attempt to permanently save the files in
     // the database by calling the Save command on the dataset and/or version.
+    @TransactionAttribute(TransactionAttributeType.NOT_SUPPORTED)
     public List<DataFile> saveAndAddFilesToDataset(DatasetVersion version, List<DataFile> newFiles) {
```

Comment:
Did we check that removing transaction here doesn't cause any problems with file replacing through UI (`ReplaceFileHandler.integrateFileWithDataset`)?

Reply:
I've checked (on three different files) and it seems to work properly.